### ISSUE TITLE
[not an issue] [enhancement] Add custom wallpapers?

### ISSUE THREAD

**Issue by purplpasta** [m4heshd/whatsapp-desktop-dark#143]
Since you managed to edit the default background to a dark one, could you maybe allow users to insert their own backrounds as png files?

Or if that is not possible, just add/change some of the already existing solid-color-backgrounds. Just some more customizablity.

(Just some quick idea how this might look like I made)
![WhatsAppDesktopCustomTheme](https://user-images.githubusercontent.com/56203839/75097526-f831c000-55ab-11ea-8ec0-9f8b9d77159b.png)

**Comment by m4heshd** [comment]
Hi @OwO-tism,
First of all your mock-up looks really good. This is a feature request is actually already possible but not in the way you describe it. It's actually not possible to change functionality on runtime because of the limitations on the renderer process set by WA devs. So it has to be an installation step.

I want you to try something. 

1. Select an image of your choice (Should be a `png`)
2. Rename it to `bg-chat-tile_9e8a2898faedb7db9bf5638405cf81ae.png`
3. Place it in WADark's `styles/$PLATFORM$/img` directory
4. Run the installation process

Now check WhatsApp to see if you have that background image. Let me know how it goes.

**Comment by purplpasta** [comment]
I tried the instructions, but no result. Not set as background by default, nor in the "Set Chat Wallpaper" list. Does the png require a specific size?

**Comment by m4heshd** [comment]
What version of WhatsApp are you on? Also judging by your mock-up, platform is Windows right?

**Comment by purplpasta** [comment]
I am on version 0.4.315, and yes, I am using Windows 8.1 32 bit (if it matters)

**Comment by m4heshd** [comment]
That's odd because it works for me. Although with the current styles that's used to modify that background image, the custom image will look awful. This to me at the moment looks like a lot of effort is needed. So we're gonna have to keep this in the bag for some other time. Sorry about that.

**Comment by purplpasta** [comment]
I just noticed that it did work (kinda). Closely look at the background here:
![image](https://user-images.githubusercontent.com/56203839/75265652-7f23aa00-57f1-11ea-96e4-42c087dac0b3.png)
(if you can't make it out, it is easier to see here)
![image](https://user-images.githubusercontent.com/56203839/75265933-ee999980-57f1-11ea-8b91-22cf760fde30.png)

It is just reeally dark

**Comment by m4heshd** [comment]
Yes. That does match the styles. I'll try to take a look in the next release.
